### PR TITLE
Added functions to the EJBCA setup to have a similar work flow as the…

### PR DESCRIPTION
**Pull request by aveenismail**
… KSP

@a-dma @klali

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -244,8 +244,8 @@ fn get_threshold_and_shares() -> (u32, u32) {
     }
 }
 
-fn object_to_file(id: u16, data: &[u8]) -> Result<String, String> {
-    let path_string = format!("./0x{:04x}.yhw", id);
+fn object_to_file(id: u16, object_type: ObjectType, data: &[u8]) -> Result<String, String> {
+    let path_string = format!("./0x{:04x}-{:?}.yhw", id, object_type);
```
Comment (a-dma) [inline src/main.rs:248]: don't rely on the Debug implementation

Comment (aveenismail) [reply src/main.rs:248]: What do you mean?

Comment (a-dma) [reply src/main.rs:248]: don't print using `{:?}`, that uses the Debug trait. Print using `{}` which uses the Display trait.

Comment (aveenismail) [reply src/main.rs:248]: I C. Ok

**Pull request closed without merging**